types/src/front/api_handlers/internal/agent_configuration.ts

```diff
@@ -202,3 +202,7 @@ export const PostOrPatchAgentConfigurationRequestBodySchema = t.type({
 export type PostOrPatchAgentConfigurationRequestBody = t.TypeOf<
   typeof PostOrPatchAgentConfigurationRequestBodySchema
 >;
+
+export type ActionAgentConfigurationType = t.TypeOf<
```

Comment:
Lol that's a bit of a lie :)

Should be `AgentActionConfigurationType`

Reply:
Fixed.